src/Core/src/Core/IButton.cs

```diff
@@ -5,5 +5,6 @@ public interface IButton : IView, IText
 		void Pressed();
 		void Released();
 		void Clicked();
+		Thickness Padding { get; }
```

Comment:
Padding is a property that will be shared between several controls: Button, ImageButton.
Create an `IPadding` interface, and implement the interface in IButton, this way we have everything ready for ImageButton.

Reply:
About this interface should I change it on `Label` as well? I didn't implement it because the LabelHandler doesn't have this interface but has the Padding property.

Reply:
For now, I added that interface on Label, let me know if that isn't required.